src/main/java/at/hannibal2/skyhanni/data/HypixelData.kt

```diff
@@ -89,11 +91,11 @@ object HypixelData {
     )
 
     /**
-     * REGEX-TEST:           §r§b§lCoop §r§f(4)
+     * REGEX-TEST: §8[§r§a§r§8] §r§bBpoth §r§6§l℻
      */
-    private val playerAmountCoopPattern by patternGroup.pattern(
+    private val playerAmountOnIslandPattern by patternGroup.pattern(
         "playeramount.coop",
-        "^\\s*(?:§.)*Coop (?:§.)*\\((?<amount>\\d+)\\)\\s*$",
+        "^§.\\[[§\\w]{6,11}] §r.*",
```

Comment:
changing this pattern from using a capture group to not using a capture group will cause NPEs for people on old versions, please change the pattern key